src/test/frameworks/next/utils.spec.ts

```diff
@@ -245,22 +245,22 @@ describe("Next.js utils", () => {
       stub.withArgs(EXPORT_MARKER).resolves(exportMarkerWithImage);
       stub.withArgs(IMAGES_MANIFEST).resolves(imagesManifest);
 
-      expect(await isUsingImageOptimization("")).to.be.true;
```

Comment:
Add 2 tests for `*client-reference-manifest.js`.
One for legacy, and another one for v`13.4.10` containing app dir.

Reply:
Added these test cases to the `isUsingNextImageInAppDirectory` tests instead as that's the function handling `client-reference-manifest.js`